callbacks.js

```diff
@@ -1,38 +1,37 @@
-function isVowel (value) {
-    vowels = 'aeiouAEIOU'
-    // if the value is a single letter string and a vowel
-    if(value.length == 1 && vowels.includes(value) ){
-        return true
-    } else {
-        return false
+    function isVowel (value) {
+        vowels = 'aeiouAEIOU'
+        // if the value is a single letter string and a vowel
+        if(value.length == 1 && vowels.includes(value) ){
+            return true
+        } else {
+            return false
+        }
     }
-}
 
-console.log(isVowel('a'))
-console.log(isVowel('basd'))
-console.log(isVowel(2))
+    console.log(isVowel('a'))
+    console.log(isVowel('basd'))
+    console.log(isVowel(2))
 
-function countValues (value) {
-    letterList = [];
-	vowelCount = 0;
-	for (let i = 0; i < value.length; i++){
-		letterList.push(value[i].split());
-	}
-	vowelList = letterList.filter(isVowel);
-	for (let j = 0; j < vowelList.length; j++){
-		vowelCount++;
-	}
-	return vowelCount;
-}
-
-console.log(countValues('asdsdgoek'))
-console.log(countValues('abcdefghijklmnopqrstuvwxyz'))
-console.log(countValues('aeiou'))
-console.log(countValues('aaaaaaaaaaa'))
+    function countValues (value) {
+        letterList = [];
+        vowelCount = 0;
+        for (let i = 0; i < value.length; i++){
+            letterList.push(value[i].split());
+        }
+        vowelList = letterList.filter(isVowel);
+        for (let j = 0; j < vowelList.length; j++){
+            vowelCount++;
+        }
+        return vowelCount;
+    }
 
+    console.log(countValues('asdsdgoek'))
+    console.log(countValues('abcdefghijklmnopqrstuvwxyz'))
+    console.log(countValues('aeiou'))
+    console.log(countValues('aaaaaaaaaaa'))
 
 
-function byVowelCount (array) {
-    return array.sort()
-}
-console.log(byVowelCount())
\ No newline at end of file
+    function byVowelCount (array) {
+        return array.sort()
```

Comment:
Ok so sort takes a function of two values: A and B (think of these as comparing adjacent values until it it doesn't change the order anymore).

```js
(valueA, valueB) => {}
```
The return value should be a boolean (truthy or falsy will work too)

What do we want compare between the two values?  How many vowels they have.

You have a function that does this.  So how would get the number of vowels for the compared values?  Then return which is bigger than the other?